src/snowflake/snowpark/dataframe.py

```diff
@@ -539,6 +541,10 @@ def __init__(
     def stat(self) -> DataFrameStatFunctions:
         return self._stat
 
+    @property
+    def transform(self) -> DataFrameTransformFunctions:
```

Comment:
`transform` is a very generic term. Is there any reason we choose it?

Reply:
"transform" is the keyword agreed upon in the meeting. It's supposed to be generic. The functions following this keyword will indicate the kind of transformation we support. 
Do you have any alternate suggestions?

Reply:
As discussed offline. Changed it to "analytics"